nvflare/fuel/f3/cellnet/cell.py

```diff
@@ -215,7 +176,7 @@ def send(self):
             f"{self.cell.get_fqcn()}: bulk sender {self.target} sending bulk size {len(messages_to_send)}"
         )
         tms = [m.to_dict() for m in messages_to_send]
-        bulk_msg = new_message(payload=tms)
+        bulk_msg = Message(None, tms)
```

Comment:
Can we make this explicit `msg = Message(headers=None, payload=tms)`?

Reply:
Keyword should be only used when there are many arguments. Using keyword arg everywhere defeats the purpose of positional arg.